chart/flux/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: "1.18.0"
-version: 1.2.0
+version: 1.2.1
```

Comment:
The release of our Helm chart is not automated, and the version is thus bumped during the release to not skip any versions.